qiskit/circuit/library/overlap.py

```diff
@@ -15,6 +15,7 @@
 from qiskit.circuit import QuantumCircuit, Gate
 from qiskit.circuit.parametervector import ParameterVector
 from qiskit.circuit.exceptions import CircuitError
+from qiskit.circuit.library import Barrier
```

Comment:
`Barrier` actually lives in `qiskit.circuit` - I don't really know why we re-export it out of `library`.